src/lib.rs

```diff
@@ -200,10 +233,17 @@ impl fmt::Display for DummyKeyHash {
     }
 }
 
-impl ToPublicKeyHash for DummyKeyHash {
-    fn to_public_key_hash(&self) -> hash160::Hash {
-        use bitcoin_hashes::hex::FromHex;
-        hash160::Hash::from_hex("0000000000000000000000000000000000000000").unwrap()
+impl ToHash160 for DummyKeyHash{
+
+    fn to_hash160(&self) -> hash160::Hash{
+        hash160::Hash::from_str("").unwrap()
```

Comment:
I wonder about the consistency requirement that hashing the output of `ToPublicKey::to_publickey` should be the same as the output of `MiniscriptKey::to_pubkeyhash` then `ToHash160::to_hash160`.

But what you've got now is fine, if we want to change it later we can. But I think we'll probably just have to be sure to document this.